packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/SelectedEntriesModal/index.js

```diff
@@ -146,15 +304,27 @@ const SelectedEntriesModal = ({ onToggle, onConfirm }) => {
   }, [rows, selectedEntries]);
 
   return (
-    <Table.Root rows={entries} defaultSelectedEntries={selectedEntries} colCount={4}>
-      <SelectedEntriesModalContent onToggle={onToggle} onConfirm={onConfirm} />
+    <Table.Root
+      rows={entries}
+      defaultSelectedEntries={selectedEntries}
+      colCount={4}
+      isLoading={isRefreshing}
+    >
+      <SelectedEntriesModalContent
+        onToggle={onToggle}
+        onConfirm={onConfirm}
+        onRefreshData={onRefreshData}
+        isRefreshing={isRefreshing}
+      />
```

Comment:
I find the SelectedEntriesModal component kind of weird since it just forwards props to another component. Maybe we can find a better solution in another PR.